README.md

```diff
@@ -143,6 +143,11 @@ VARIABLE_WITH_DEFAULT: !var:default='defaultvalue' path/to/variable
 
 * `-f <path>` specify a location to a secrets.yml file, default 'secrets.yml' in current directory.
 
+* `-u, up` searches for secrets.yml going up, starting from the current working directory
+
+    Stops at the first file found or when the root of the current file system is reached.
```

Comment:
This needs to be split into multiple lines to meet the line length requirement of 80 characters

Reply:
Done. (Note that this file has many lines > 80 chars).